Patches/ExilePatch.cs

```diff
@@ -170,33 +170,27 @@ static void WrapUpPostfix(GameData.PlayerInfo exiled)
         if (Witch.IsEnable) 
             Witch.RemoveSpelledPlayer();
 
-        if (HexMaster.IsEnable)
-            HexMaster.RemoveHexedPlayer();
-
+        if (Swapper.Vote.Any() && Swapper.VoteTwo.Any())
+        {
+            foreach (var swapper in Main.AllAlivePlayerControls)
+            {
+                if (swapper.Is(CustomRoles.Swapper))
+                {
+                    Swapper.Swappermax[swapper.PlayerId]--;
+                    Swapper.Vote.Clear();
+                    Swapper.VoteTwo.Clear();
+                    Main.SwapSend = false;
+                }
+            }
+        }
+        
```

Comment:
Is the code below necessary then?
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/48de23ac-7dd2-4d5a-af6a-d4fdd775c5ea)

Reply:
WTF does this code do.